accessibility-checker/src/lib/ACConfigLoader.js

```diff
@@ -106,6 +106,9 @@ async function processACConfig(ACConfig) {
         try {
             ruleArchiveParse = await new Promise((resolve, reject) => {
                 request.get(ruleArchiveFile, function (error, response, body) {
+                    if (error) {
+                        throw error;
```

Comment:
Since this is inside of a Promise, instead of `throw error`, it should be `reject(error)`. The `await` will convert the Promise rejection into a throw that will be caught by the try block.

Reply:
Thanks, this has been updated 👍